src/components/CategoryFilter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const OPTIONS = [
+  { name: 'All', id: 1 },
+  { name: 'Action', id: 2 },
+  { name: 'Biography', id: 3 },
+  { name: 'History', id: 4 },
+  { name: 'Horror', id: 5 },
+  { name: 'Kids', id: 6 },
+  { name: 'Learning', id: 7 },
+  { name: 'Sci-Fi', id: 8 },
+];
+
+const CategoryFilter = props => {
+  const { handleChange } = props;
```

Comment:
- [ ] **[OPTIONAL]** Good job using object destructing here. We can also directly destruct objects inside function arguments:

```js
const CategoryFilter = ({ handleChange }) => {
```

This will save you a few lines of code.